app/Http/Controllers/Api/DailyMenuController.php

```diff
@@ -3,88 +3,64 @@
 namespace App\Http\Controllers\Api;
 
 use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use App\Http\Controllers\Controller;
+use App\DailyMenu;
 
-class DailyMenuController extends ApiController
+class DailyMenuController extends Controller
 {
     /**
-     * Display a listing of the resource.
+     * The Daily Menu implementation.
      *
-     * @return \Illuminate\Http\Response
+     * @var DailyMenu
      */
-    public function index()
-    {
-        //
-    }
+    protected $dailyMenu;
 
     /**
-     * Show the form for creating a new resource.
+     * Create a new controller instance.
      *
-     * @return \Illuminate\Http\Response
-     */
-    public function create()
-    {
-        //
-    }
-
-    /**
-     * Store a newly created resource in storage.
+     * @param DailyMenu $dailyMenu instance of DailyMenu
      *
-     * @param \Illuminate\Http\Request $request request create
-     *
-     * @return \Illuminate\Http\Response
+     * @return void
      */
-    public function store(Request $request)
+    public function __construct(DailyMenu $dailyMenu)
     {
-        $request = $request;
+        $this->dailyMenu = $dailyMenu;
     }
 
     /**
-     * Display the specified resource.
-     *
-     * @param int $id id daily menu
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function show($id)
-    {
-        $id = $id;
-    }
-
-    /**
-     * Show the form for editing the specified resource.
-     *
-     * @param int $id id daily menu
+     * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
-    public function edit($id)
+    public function index()
     {
-        $id = $id;
+        $dailyMenus = $this->dailyMenu->search()->select('date')->distinct()->orderBy('date', 'DESC')->paginate(DailyMenu::ITEMS_PER_PAGE);
+        return response()->json(collect(['success' => true])->merge($dailyMenus));
     }
 
     /**
-     * Update the specified resource in storage.
+     * Display the specified resource.
      *
-     * @param \Illuminate\Http\Request $request request update
-     * @param int                      $id      id daily menu update
+     * @param string $date The date to get menu to show
      *
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function show($date)
     {
-        $request = $request;
-        $id = $id;
-    }
+        $error = __('Has error during access this page');
```

Comment:
You should make `$error` before response error. In case function response success, this variable doesn't be to use.